packages/framework-provider-local-infrastructure/src/index.ts

```diff
@@ -41,6 +44,11 @@ export const Infrastructure = (): ProviderInfrastructure => {
       const userProject: UserApp = require(path.join(process.cwd(), 'dist', 'index.js'))
       const graphQLService = new GraphQLService(userProject)
       router.use('/graphql', new GraphQLController(graphQLService).router)
+      if (rockets && rockets.length > 0) {
+        rockets.map((rocket) => {
```

Comment:
Suggestion: this could be a `.forEach`as you are not transforming the array 😉

Reply:
Good catch!